packages/shared/src/components/onboarding/FeedReadyMessage.tsx

```diff
@@ -15,6 +16,11 @@ export const FeedReadyMessage = ({
   className,
 }: FeedReadyMessageProps): ReactElement => {
   const { user } = useContext(AuthContext);
+  const { alerts } = useAlertsContext();
+
+  if (alerts?.filter) {
+    return null;
+  }
```

Comment:
Should this be under feature flag, does it apply for control?

Reply:
Was thinking to apply it to everything. In case you somehow stumble upon https://app.daily.dev/?welcome=true if you have an account, but haven't created your feed, you shouldn't really see the "your feed is ready message". And it probably shouldn't be possible to view it on the other feeds as well 😈

- https://app.daily.dev/popular?welcome=true
- https://app.daily.dev/squads/watercooler?welcome=true

Reply:
Ah gotcha, makes sense. One edge case comes to mind though. Do we set `alerts.filter` in onboarding so then it might happen that we pass `welcome=true` but because you already set `alerts.filter` it would not show? That would happen in control I guess.

Reply:
So just double check if you can 🙏

Reply:
I think that setting the `alerts.filter` happens on account creation. And that alert should be removed when you choose a tag (be it onboarding or anywhere else).
https://github.com/dailydotdev/apps/blob/2f390ae941d70f31ddf5870cd16c5f248e4d86b3/packages/shared/src/hooks/useTagAndSource.ts#L88-L90

I just added so that in this experiment, we don't clear the filter alerts until the click "create feed".

Does that answer your edge case?